libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/processes-table/dh-processes-table.component.scss

```diff
@@ -78,4 +79,7 @@ mat-row {
 .has-details-errors-cell-header,
 .has-details-errors-cell {
   flex: 0 1 calc(var(--cell-padding) * 3);
+  div {
+    width: 100%;
+  }
```

Comment:
question: is there any reason for wrapping the icon in a extra `div` with class `watt-text-s`? If not, consider removing the `div` and replace the styling above with `justify-content: center`. This will also center the icon.